src/awslambda/extract/bobaedream_exec.py

```
@@ -175,12 +175,24 @@ def parse_post_meta(post, post_meta):
     except Exception as e:
         print(f"[ERROR] 조회수 파싱 실패: {e}")
         view = -999
-    like = count_group_em[2].text
-    date_str = count_group.text
-    date_str = date_str.split('|')[2].strip()
-    posting_datetime = clean_date_string(date_str)
+    try:
+        like = count_group_em[2].text
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)